pubtools/pulplib/_impl/model/unit/erratum.py

```diff
@@ -374,6 +374,15 @@ class ErratumUnit(Unit):
     )
     """A list of package collections associated with the advisory."""
 
+    container_list = pulp_attrib(
+        type=list,
+        pulp_field="pulp_user_metadata.container_list",
+        converter=frozenlist_or_none_converter,
+        default=None,
+        validator=optional_list_of(str),
+    )
+    """A list of package collections associated with the advisory."""
```

Comment:
Doc string is wrong.  Could it please be updated for accuracy?

It should mention the format of the strings or at least give an example or two.